pkgs/os-specific/linux/sgx-sdk/samples.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv
+, sgx-sdk
+, which
+}:
+let
+  buildSample = name: stdenv.mkDerivation rec {
+    inherit name;
+
+    src = sgx-sdk.out;
+    sourceRoot = "${sgx-sdk.name}/share/SampleCode/${name}";
+
+    buildInputs = [
+      sgx-sdk
+    ];
+
+    buildFlags = [
+      "SGX_MODE=SIM"
```

Comment:
Does this run on amd cpus or on intel CPUs without SGX as well?

Reply:
Yes, this should work. I didn't test it though as I don't have a machine with an AMD CPU available.

Reply:
Works on AMD EPYC.